### ISSUE TITLE
gm-installer old version

### ISSUE THREAD

**Issue by MrWildanMD** [codership/galera-manager-support#118]
how to install gm-installer old version because i need to testing with mariadb 10.3.x

**Comment by ayurchen** [comment]
I don't think we ever supported MariaDB 10.3. In any case it is EOL and has such an outdated Galera library that a Galera cluster based on MariaDB should not be used.
